javascript-loops/main.js

```diff
@@ -0,0 +1,87 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+
+  return numbers;
+}
+console.log('Get numbers to ten ()', getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+console.log('even numbers to twenty', getEvenNumbersToTwenty());
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+
+  while (count <= times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+console.log('repeated word:', repeatWord('hello', 30));
+
+function logEachCharacter(string) {
+  for (let i = 0; i < string.length; i++) console.log(string[i]);
```

Comment:
Works, but I always recommend using curly brackets. You can have some unintended consequences if you don't use them

```
for (let i = 0; i < string.length; i++) {
   console.log(string[i]);
}
```